### ISSUE TITLE
circuit breaker returns null when fallback is specified

### ISSUE THREAD

**Issue by ashish-s** [jhalterman/failsafe#75]
when i specify a fallback action in circuitbreaker version 1.0.0, it fails to returns a valid value of the main operation. This used to work in 0.9.2

```
@Test
 public void testThatFailSafeIsBrokenWithFallback() throws Exception {
        final CircuitBreaker builderBreaker = new CircuitBreaker();
        CircuitBreaker breaker = builderBreaker.withFailureThreshold(10, 100).withSuccessThreshold(2)
                                               .failOn(DEFAULT_FAIL_ON_EXCEPTION_LIST)
                                               .withDelay(100, TimeUnit.MILLISECONDS);
        ScheduledExecutorService service = Executors.newSingleThreadScheduledExecutor();
        Integer result = Failsafe.with(breaker).with(service).withFallback((a, b) -> 999)
                                 .future(() -> CompletableFuture.completedFuture(223)).get();

        assertNotNull(result); // this should return 223
    }
```

**Comment by AlexPayment** [comment]
Tested with every versions from 0.9.2 to 1.0.0 and it started in 0.9.3.

**Comment by jhalterman** [comment]
Thanks for reporting this!

**Comment by jhalterman** [comment]
Just released 1.0.2 which includes this fix.

**Comment by ashish-s** [comment]
thanks @jhalterman for the  response...
